### PR TITLE
Fix shuffle error

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR fixes the error in writing the shuffle status to the pandas df during training and testing.

### CODE REVIEW

File: 4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/Multi_Classifier_Call.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #SBATCH --partition=amem
 #SBATCH --qos=mem
 #SBATCH --account=amc-general
-#SBATCH --time=72:00:00
+#SBATCH --time=36:00:00
```
Comment (MattsonCam) [inline 4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/Multi_Classifier_Call.sh:9]: Didn't know slurm syntax was specified like a bash comment, not sure if I like that

File: 4.sc_Morphology_Neural_Network_MLP_Model/figures/Multi_Class/MultiClass_MLP/PBMC/Accuracy_graph.png
Comment (MattsonCam) [inline 4.sc_Morphology_Neural_Network_MLP_Model/figures/Multi_Class/MultiClass_MLP/PBMC/Accuracy_graph.png:1]: It looks like some of the figures changed, not sure if this was expected or known, but wanted to draw your attention.

File: 4.sc_Morphology_Neural_Network_MLP_Model/figures/Multi_Class/MultiClass_MLP/PBMC/confusion_matrix_graph_MultiClass_MLP_treatment_hold_out.png
Comment (MattsonCam) [inline 4.sc_Morphology_Neural_Network_MLP_Model/figures/Multi_Class/MultiClass_MLP/PBMC/confusion_matrix_graph_MultiClass_MLP_treatment_hold_out.png:1]: Maybe it is the way I am viewing it, but I think some of these figures have some formatting errors

File: 4.sc_Morphology_Neural_Network_MLP_Model/trained_models/architectures/Multi_Class/PBMC/Multi_Class_MultiClass_MLP.json [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-        0.2574816738272465,
-        0.11278714608165805
+        0.2296523795897401
     ],
-    "n_layers": 2,
-    "optimizer": "Adam",
-    "learning_rate": 0.0004056418371923762
+    "n_layers": 1,
+    "optimizer": "SGD",
+    "learning_rate": 0.09362772149064716
```
Comment (MattsonCam) [inline 4.sc_Morphology_Neural_Network_MLP_Model/trained_models/architectures/Multi_Class/PBMC/Multi_Class_MultiClass_MLP.json:10]: Oh, I see, this is probably why the figures look different now

File: 4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/train_multiclass_model.py
Comment (MattsonCam) [inline 4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/train_multiclass_model.py:1]: This is a lot of code, consider distributing the code in this python file to multiple files.

**Review by MattsonCam — APPROVED** [review]
Hey @MikeLippincott, I'll go ahead and approve. I think there are some minor changes you may want to make. Overall, good job!

File: 4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/Multi_Classifier_Call.sh (hunk 1 above)
Comment (MikeLippincott) [reply 4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/Multi_Classifier_Call.sh:9]: Haha! Yes! I love this comment @MattsonCam . It feels...off. But nerveless, this is how it is done.

File: 4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/train_multiclass_model.py
Comment (MikeLippincott) [reply 4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/train_multiclass_model.py:1]: I referenced this in a new issue
